### PR TITLE
Added More CLI and Config elements

### PR DESCRIPTION
**Pull request by AliabbasMerchant**
@YourTechBud Done

### CODE REVIEW

File: config/constraints.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	
+	// Load config from file
+	conf, err := LoadConfigFromFile(utils.DefaultConfigFilePath)
+	if err != nil {
+		return err
+	}
+
+	rep, err := strconv.ParseInt(replicas, 10, 32)
+	if err != nil {
+		return errors.New("Number of Replicas Must be an int32 Value")
```
Comment (YourTechBud) [inline config/constraints.go]: Must be an integer value.

Comment (YourTechBud) [reply config/constraints.go]: Capitalise only the first letter of the sentence. Fix this for all messages

File: config/constraints.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	
+	// Load config from file
+	conf, err := LoadConfigFromFile(utils.DefaultConfigFilePath)
+	if err != nil {
+		return err
+	}
+
+	CPU, err := strconv.ParseFloat(cpu, 32)
+	if err != nil {
+		return errors.New("CPU Limit Must be a float32 Value")
```
Comment (YourTechBud) [inline config/constraints.go]: Decimal?

File: config/constraints.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	}
+	if CPU <= 0 {
+		return errors.New("CPU Limit must be a Positive Fraction")
```
Comment (YourTechBud) [inline config/constraints.go]: Decimal?

File: config/constraints.go [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+	
+	// Load config from file
+	conf, err := LoadConfigFromFile(utils.DefaultConfigFilePath)
+	if err != nil {
+		return err
+	}
+
+	mem, err := strconv.ParseInt(memory, 10, 64)
+	if err != nil {
+		return errors.New("Memory Limit must be an int64 value")
```
Comment (YourTechBud) [inline config/constraints.go]: Integer value

File: config/env.go [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+	}
+
+	// Load config from file
+	conf, err := LoadConfigFromFile(utils.DefaultConfigFilePath)
+	if err != nil {
+		return err
+	}
+
+	_, ok := conf.Env[name]
+	if !ok {
```
Comment (YourTechBud) [inline config/env.go]: Override the environment variable if it already exists. No need to throw an error

File: config/env.go [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+		return errors.New(name + " does not exist")
+	}
+
+	delete(conf.Env, name)
+	return StoreConfigToFile(conf, utils.DefaultConfigFilePath)
+}
+
+// SetEnvVar sets the environment variable value
+func SetEnvVar(name, value string) error {
```
Comment (YourTechBud) [inline config/env.go]: Lets just keep set which works both as update and add

File: config/expose.go [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+		return err
+	}
+
+	return StoreConfigToFile(conf, utils.DefaultConfigFilePath)
+}
+
+func addExpose(conf *model.Deploy) error {
+	expose := new(model.Expose)
+	path := ""
+	err := survey.AskOne(&survey.Input{Message: "path:"}, &path, survey.Required)
```
Comment (YourTechBud) [inline config/expose.go]: This should be prefix

File: config/port.go [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+				exists = true
+				break
+			}
+		}
+	}
+	if exists {
+		return errors.New("Port " + name + " already exists")
+	}
+	port := &model.Port{Name: &name}
+	err = askPortInfo(port, "", "")
```
Comment (YourTechBud) [inline config/port.go]: Let the default port be 8000 and default protocol be TCP

**Comment by AliabbasMerchant** [comment]
@YourTechBud Done

**Pull request merged**